air.js.org

SisyphusZheng opened the pull request:
- [✔] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [✔] I have read and accepted the 
- The site content can be seen at https://openwebstudio.github.io/AirJS/

> The site content is a documentation site for my JavaScript framework, AirJS, showcasing its features, API references, and usage examples.

MattIPv4 reviewed (changes requested):
Hi there!

❌ Before we can accept your js.org domain request, your site must have reasonable content.
Based on https://openwebstudio.github.io/AirJS/ your site currently does not contain what we would consider to be reasonable content to grant a request.

<img width="1680" alt="image" src="https://github.com/user-attachments/assets/6b51a681-8435-4ed1-8964-ece423a52257" />

Please see the wiki article: https://github.com/js-org/js.org/wiki/No-Content 📖

MattIPv4 commented:
Looking at https://github.com/openwebstudio/AirJS/blob/gh-pages/build/index.html, I suspect you want to be outputting what is currently in `build/` into the root of `gh-pages`, or update your repository configuration to be serving from the `build/` directory instead of the root.

SisyphusZheng commented:
Hi JS.ORG maintainers,

I have updated my site to include reasonable content as per your guidelines. 
The updated site is live at: [https://openwebstudio.github.io/Air-Docs/](https://openwebstudio.github.io/Air-Docs/).
[
![screenshot-1737010970235](https://github.com/user-attachments/assets/a08f45ef-395c-422d-9b26-8dbde42a9661)
](url)
Key improvements:
- Added a detailed description of AirComponents and AirJS in the future.
- Included interactive examples showcasing core functionality.
- Enhanced page design for better user experience.

Please kindly review the updated content and reconsider my subdomain request.

Thank you for your time!

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```air.js.org``` to make it work.
I'll merge after we have "contact". Please keep this PR open.

indus commented:
Because you are using a workflow to deploy the page, additional steps may be required to make the CNAME work.
FYI: https://github.com/js-org/js.org/pull/7853#issuecomment-1336537858 / https://github.com/js-org/js.org/pull/8532#issuecomment-1732508431

SisyphusZheng commented:
Hi JS.ORG maintainers,
I have resolved all the issues mentioned earlier:
Added a CNAME file with air.js.org to the deployment directory.
Fixed the deployment path, and the site is now correctly serving the intended content.
The updated site is live at https://air.js.org.
Please review the changes and proceed with merging the PR.
![screenshot-1737038453575](https://github.com/user-attachments/assets/ebf4df7d-574b-4d0c-8f4c-c8094746c72c)

Thank you for your support!
